### PR TITLE
extend testing timeout 

### PR DESCRIPTION
**Pull request by jia-jerry**
**How to categorize this PR?**

/area quality
/kind test
/platform alicloud

**What this PR does / why we need it**:
Integration test failed from time to time in the past due to timeout(60 mins).

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```other operator

```

### CODE REVIEW

File: .test-defs/infrastructure-test.yaml [hunk 1]
```diff
@@ -14,6 +14,6 @@ spec:
       --kubeconfig=$TM_KUBECONFIG_PATH/testmachinery.config
       --access-key-id=$ACCESS_KEY_ID
       --access-key-secret=$ACCESS_KEY_SECRET
-      --region=$REGION
+      --region=cn-shanghai
```
Comment (rfranzke) [inline .test-defs/infrastructure-test.yaml:17]: Why is this changed here? Shouldn't we rather specify a different region in the test invocation?

Comment (jia-jerry) [reply .test-defs/infrastructure-test.yaml:17]: I just do a test, If it is faster in Shanghai, I will set in in a different region. BTW, it seems 1.5 h is still not safe enough. Here is image encryption time point for your PR. It took almost 1.5 h.


encrypt_image_ubuntu_20-04ALIYUN::ROS::Stack | c89ca2e8-2eec-4105-8ee2-7d1098... | Created | Stack CREATE completed successfully | Dec 21, 2021, 16:44:08
-- | -- | -- | -- | --
CopyImageALIYUN::ECS::CopyImage | m-gw81rit1umguzxbxjr1a | Created | state changed | Dec 21, 2021, 16:44:07
CopyImageALIYUN::ECS::CopyImage | - | Creating | state changed | Dec 21, 2021, 15:16:35
encrypt_image_ubuntu_20-04ALIYUN::ROS::Stack | c89ca2e8-2eec-4105-8ee2-7d1098... | Creating | Stack CREATE started | Dec 21, 2021, 15:16:35

Comment (jia-jerry) [reply .test-defs/infrastructure-test.yaml]: I got your point. I found the right place to change the region.

**Comment by jia-jerry** [comment]
I will create another PR to fix ROS issue.
